notes/klustered-teams/episode-3/skyscanner/README.md

```diff
@@ -0,0 +1,36 @@
+# Kluster 000
+
+## Discovered Symptoms
+
+
+## Contributing Factors
+
+
+## Notes from Kluster Breaker
+
+### APIServer Break
+
+We modified the core `v1.` [apiservice](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/#register-apiservice-objects) object to point to the `kubernetes` service in the `default` namespace, resulting in the APIServer being unable to ever successfully break out of a loop and respond as all API requests got trapped in a loop. The reason restarting the APIServer was enough to fix this was that the built-in aggregator server which manages aggregation of APIs within kubernetes rebuilds this list on startup from the API resources defined within the Kube-APIServer.
+
```

Comment:
Might be worth mentioning that this did work on older versions of kubernetes, but didn't here.

```suggestion
Before breaking this, we had only changed `v1.apps` to point to a fake `kubrenetes` (with typo) service in the default namespace. In older versions of kubernetes, we found that pointing to the real `kubernetes` service worked. It goes to show that there are ways you can apply objects that break your cluster (sometimes beyond repair).
```